scipy/stats/tests/test_mstats_basic.py

```diff
@@ -550,6 +550,20 @@ def test_trimmedmean(self):
         assert_almost_equal(mstats.trimmed_mean(data,(0.1,0.1)), 343, 0)
         assert_almost_equal(mstats.trimmed_mean(data,(0.2,0.2)), 283, 0)
 
+    def test_trimmedvar(self):
+        data = ma.array([77, 87, 88, 114, 151, 210, 219, 246, 253, 262,
+                         296, 299, 306, 376, 428, 515, 666, 1310, 2611])
```

Comment:
What is the source of this data?
If the data is arbitrary, why not generate the data at random so it is clear that there is nothing special about the data?

Reply:
I don't know the source of it. I borrowed it from the neighboring functions which test `trimmed_mean` and some other functions. However, now that you mention it, I too think that randomly generated data would be better.